### PR TITLE
feat: freeze config lib [PLAT-3319]

### PR DESCRIPTION
**Pull request by alexsupp**
- freeze config lib at 3.3.7

### CODE REVIEW

**Review by hershmire — APPROVED** [review]
Looks good to me. Let's check in with @prescottprue first.

File: README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Used by other presets
 - Skips `faker` and `@types/fake` updates since it is no longer supported
 - Ignores Side Inc. private docker image updates (registry auth not yet setup) [PLAT-1660](https://residenetwork.atlassian.net/browse/PLAT-1660)
 - Support for Side Inc. private NPM dependencies
+- Skips `config` updates after 3.3.7 due to breaking change
```
Comment (prescottprue) [inline README.md:67]: Can we call out what the breaking change is? And/or link to it? or a break it caused

Comment (hershmire) [reply README.md:67]: @alexsupp link around details on the regression is here https://github.com/node-config/node-config/issues/703.

Comment (alexsupp) [reply README.md]: @prescottprue linked

**Review by hershmire — APPROVED** [review]

**Pull request merged**